packages/mwp-app-server/src/util/getPlugins.js

```diff
@@ -91,6 +93,20 @@ export function getCsrfPlugin(electrodeOptions) {
 		},
 	};
 }
+/**
+ * We are using Blankie as our Content Security Policy (CSP) plugin 
+ * Which uses Scooter to detect the user agent and apply the appropriate
+ * CSP header, usually Content-Security-Policy but some older browsers are
+ * slightly different. A CSP compatible browser will use the header to ignore
+ * scripts not whitelisted in our policy header.
+ * https://github.com/nlf/blankie 
+ */
+export function getCspPlugin(options) {
+	return {
```

Comment:
Looking good so far. In order to add the remaining headers, you'll need to register an `onPreResponse` handler that can call [`request.response.header(name, value)`](https://hapijs.com/api#-responseheadername-value-options)

This essentially means [writing a plugin](https://hapijs.com/tutorials/plugins?lang=en_US) with a `register` function that does something like

```js
server.ext('onPreResponse', (request, h) => {
  request.response.header('Foo', 'Bar');
  request.response.header('Baz', 'Qux');
  ...
});
```

Consider whether you would like to combine this with 'blankie' internally, rather than registering it as a separate plugin, e.g. in the plugin's `register` method, you could have something like

```js
export function register(server: HapiServer, options: ?Object) {
  server.ext('onPreResponse', (request, h) => {
    request.response.header('Foo', 'Bar');
    request.response.header('Baz', 'Qux');
    ...
  });
  return blankie.register(server, options);
```

That might not be a good idea, but something to think about.